src/utils/SavedSettings.ts

```diff
@@ -286,6 +286,7 @@ export interface VendorSettings {
     staff: boolean
     wand: boolean
   }
+  gems?: string[]
```

Comment:
I left this optional for backcompat, but if you'd prefer to force there to be an empty list value, I can add a migration instead

Reply:
Backward comp is good, this is very reasonable